whitepaper/whitepaper.md

```diff
@@ -1437,6 +1440,47 @@ provided here for ease of understanding:
 
 <img width="700" src="./plugin-consortium-manual-bootstrap-sequence-diagram.png">
 
+## 5.7 Deployment Scenarios
+
+There's a near-infinite possible deployment scenarios which is intended to be one of the benefits of using Cactus where
+you are not constrained by strong opinions imposed upon you by the framework but instead can tailor it to your needs
+while still getting value out of it.
+There's a set of building blocks (members, nodes, API server processes, plugin instances) that you can use when defining (founding) a consortium and these building blocks relate to each other in a way that can be expressed with an entity relationship diagram which can be seen below.
+The composability rules can be deducted from how the diagram elements (entities) are connected (related) to each other, e.g. the API server process can have any number of plugin instances in it and a node can contain any number of API server processes, and so on until the top level construct is reached: the consortium.
```

Comment:
_The composability rules can be deducted from how the diagram elements (entities) are connected (related) to each other, e.g. the API server process can have any number of plugin instances in it and a node can contain any number of API server processes, and so on until the top level construct is reached: the consortium._ should be replaced with following texts:
The trust relationship between the CACTUS system and each Ledger should be pre-configured and stored at the TrustedLedger Repository. 
TrustedLedgerRepository also manages the Validator's public key information which is used to sign the valid transactions.
The operation API of TrustedLedgerRepository can be chosen as  CACTUS Plugin.

Reply:
> _The composability rules can be deducted from how the diagram elements (entities) are connected (related) to each other, e.g. the API server process can have any number of plugin instances in it and a node can contain any number of API server processes, and so on until the top level construct is reached: the consortium._ should be replaced with following texts:
> The trust relationship between the CACTUS system and each Ledger should be pre-configured and stored at the TrustedLedger Repository.
> TrustedLedgerRepository also manages the Validator's public key information which is used to sign the valid transactions.
> The operation API of TrustedLedgerRepository can be chosen as CACTUS Plugin.

@sfuji822 Please see https://github.com/hyperledger/cactus/pull/331#discussion_r537955087 first!